src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -299,6 +304,17 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
         // class itself to work with the generalized ConfigStore.
         const l1Token = args.key;
 
+        // For now, we'll need to check if this value is undefined. In the
+        // future, we should make this a required field.
+        if (parsedValue.uba !== undefined) {
```

Comment:
Can we skip this parsing step in prod while continuing to be able to unit test it? I ask because we should push the config store changes on chain without fearing the existing bots crashing as soon as it tries to parse the `uba` string

Reply:
Do we have an ENV variable currently in place that can tell if the system is in prod or not?

Reply:
we do not but I would opt to hardcode the skip so that we can test locally and open a PR to remove the hardcoded skip

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/280/commits/8b5f6e9280008df22c2ab4b2e35fc3b53ead276f